### ISSUE TITLE
Incorporation of links into error messages

### ISSUE THREAD

**Issue by VisLab** [hed-standard/hed-python#730]
To facilitate putting links to the appropriate section of the specification in the error messages, I have added a file:

[hed_error_codes.json](https://github.com/hed-standard/hed-specification/blob/master/tests/hed_error_codes.json) in the specification.

which I assume both the Python and JavaScript tools would store internally (not look up on the web).

The idea is that when creating an error message for display, if the error code is listed in this file, the error message would contain a link.

**Example:**
An error with code: `HED_WIKI_SEPARATOR_INVALID` and message: "Required section separator 'EndSchema' not found in file" would display as:

[HED_WIKI_SEPARATOR_INVALID](https://hed-specification.readthedocs.io/en/latest/Appendix_B.html#wiki-separator-invalid): Required section separator 'EndSchema' not found in file.

@IanCa @happy5214

**Comment by IanCa** [comment]
I agree on directly including it.  Alternatively could potentially try to have the hedtools installer pull it from hed-specification when we compile the installer, but that seems like it might get messy to me, especially for hed developer environments.

What format for output are you thinking that supports inline linking?  Just converting to some kind of restructured text after/instead of the current text printing?

eg: 
`HED_WIKI_SEPARATOR_INVALID <https://hed-specification.readthedocs.io/en/latest/Appendix_B.html#wiki-separator-invalid>`_: Required section separator 'EndSchema' not found in file.

**Comment by VisLab** [comment]
How about:
HED_WIKI_SEPARATOR_INVALID: Required section separator 'EndSchema' not
found in file. [See
https://hed-specification.readthedocs.io/en/latest/Appendix_B.html#wiki-separator-invalid
.]

**Comment by VisLab** [comment]
Maybe we should output in HTML rather than txt or give that as an option?

**Comment by IanCa** [comment]
HTML would work...I have no strong opinion as I'm usually looking at these errors in pycharm

**Comment by VisLab** [comment]
I think we should just go with a raw text reference at the end of the message with a See...

**Comment by VisLab** [comment]
get_printable_issue_string now as a show link
get_printable_issue_string_html always shows the link.  Currently the showing the link is not used --- should it be in the web code?
